conf/machine/include/imx8ulp-evk.inc

```diff
@@ -0,0 +1,44 @@
+MACHINEOVERRIDES =. "mx8:mx8ulp:"
```

Comment:
I believe this should be just `MACHINEOVERRIDES =. "mx8ulp"`.

That said, when I grep I see the same pattern for a few more SOCs.

```
conf/machine/imx7dsabresd.conf:MACHINEOVERRIDES =. "mx7:mx7d:"
conf/machine/imx7ulpevk.conf:MACHINEOVERRIDES =. "mx7:mx7ulp:"
conf/machine/imx8qm-mek.conf:MACHINEOVERRIDES =. "mx8:mx8qm:"
```

@otavio, can you please confirm that `mx8` is not needed here?

Reply:
It is. It is meant to reuse the `mx8` common override.

Reply:
If that is the case, then it seems surprising we don't see the pattern more.

Reply:
Isn't `mx8` unused since we changed all the overrides to `mx8-*-bsp`.